Assets/Scripts/P2PNetcodeSample/Networking/EOSTransport.cs

```diff
@@ -544,8 +548,12 @@ private ProductUserId GetUserId(ulong clientId)
             }
         }
 
-        // Returns the ClientId corresponding to a given ProductUserId
-        private ulong GetClientId(ProductUserId userId)
+        /// <summary>
+        /// Returns the ClientId corresponding to a given ProductUserId
+        /// </summary>
+        /// <param name="userId">ProductUserId to use to search for the corresponding ClientId</param>
+        /// <returns></returns>
+        public ulong GetClientId(ProductUserId userId)
```

Comment:
As mentioned in [#859](https://github.com/PlayEveryWare/eos_plugin_for_unity/issues/859#issuecomment-2307539253), making this public would be problematic because the client ids are not the same as the NetworkManager and would probably add additional confusion.